common/ansi_filter.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef COMMON_ANSI_FILTER_H_
+#define COMMON_ANSI_FILTER_H_
+
+#include <string>
+
+namespace cdc_ft {
+namespace ansi_filter {
+
+// Removes ANSI escape sequences from |str|.
```

Comment:
Nit: The [style guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules) suggests to not abbreviate by deleting letters within a word.

Reply:
Discussed offline, we're still using 80 chars, so it often hurts readability. We only use abbreviations for common stuff like str(ing), var(iable) etc.